### DISCUSSION TITLE
Knob midi controller

### DISCUSSION THREAD

**Discussion by klez32** [ElectroTechnique/electrotechnique.github.io#15] [Software Issues]
Hi,this is not really a software issue, but no other category choice.
It's possible to use tsynth as a midi controller when is connect via usb?So it could be use in a daw with knob association like midi controller

Thanks in advice

**Comment by Electrotechnique** [comment]
It can output MIDI CC messages from the knobs from the MIDI Out DIN socket. This is for recording control adjustments on DAWs. The MIDI CC value is fixed however, but the MIDI channel can be set in Settings. The User Manual gives the MIDI CC values for each knob.
